packages/sdk-browser/CHANGELOG.md

```diff
@@ -1,5 +1,14 @@
 # @turnkey/sdk-browser
 
+## 1.2.4
+
+### Patch Changes
+
+- e2f2e0b: updates to sdk-browser, sdk-server and http
```

Comment:
We can remove this line if the only real change to `sdk-browser` is the `http` bump